client/packages/system/src/Patient/Components/ChipTableCell.tsx

```diff
@@ -23,6 +23,7 @@ export const ChipTableCell = <T extends RecordWithId>({
         alignItems: 'center',
         display: 'flex',
         flexWrap: 'wrap',
+        gap: 1,
```

Comment:
Side tracked: Noticed that the chips were a little too close to each other

Reply:
Not sure if that makes it better, e.g. that was it before:

![image](https://github.com/msupply-foundation/open-msupply/assets/88299195/add53fd0-4263-4ce1-84bd-5a072b679200)

and now it is:

![image](https://github.com/msupply-foundation/open-msupply/assets/88299195/56ee2485-9709-46bf-aaed-2bc74343f730)

You could add a spacing on top and bottom but the reason there was no gap was to make it more compact, i.e. minimize the line height... If you don't have a better idea, I am tending to leave it as it was before to be more space efficient. Or only add a 1px spacing?